internal/cmd/local/local/install_test.go

```diff
@@ -194,182 +171,16 @@ func TestCommand_Install(t *testing.T) {
 		t.Fatal(err)
 	}
 
-	if err := c.Install(context.Background(), InstallOpts{}); err != nil {
+	installOpts := &InstallOpts{HelmValuesYaml: valuesYaml}
+	if err := c.Install(context.Background(), installOpts); err != nil {
 		t.Fatal(err)
 	}
 }
 
-func TestCommand_Install_HelmValues(t *testing.T) {
```

Comment:
After refactoring the building of `InstallOpts`, this test became a dupe of `TestCommand_Install` because `Install()` doesn't build any values YAML now, it receives the fully formed values YAML from the caller.